### PR TITLE
[TM-853] Add download functionality for single and complete geojsons

### PR DESCRIPTION
**Pull request by egrojMonroy**
https://gfw.atlassian.net/browse/TM-853

### CODE REVIEW

File: app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php [hunk 1]
```diff
@@ -381,37 +381,105 @@ public function validateEstimatedArea(Request $request)
...
-        $features = [];
-
-        foreach ($polygons as $polygon) {
-            $coordinates = json_decode($polygon->geojson)->coordinates;
+        try {
+            $uuid = $request->query('uuid');
+
+            $polygonGeometry = PolygonGeometry::where('uuid', $uuid)
+            ->select(DB::raw('ST_AsGeoJSON(geom) AS geojsonGeom'))
+            ->first();
```
Comment (roguenet) [inline app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php:391]: These two lines should be indented. I'm surprised the linter didn't change it 🤷🏻

File: app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php [hunk 2]
```diff
@@ -381,37 +381,105 @@ public function validateEstimatedArea(Request $request)
...
+    {
+        try {
+            $siteUuid = $request->query('uuid');
+            $polygonsUuids = SitePolygon::where('site_id', $siteUuid)->pluck('poly_id');
+            $features = [];
+            foreach ($polygonsUuids as $polygonUuid) {
+                $feature = [];
+                $polygonGeometry = PolygonGeometry::where('uuid', $polygonUuid)
+                ->select(DB::raw('ST_AsGeoJSON(geom) AS geojsonGeom'))
+                ->first();
```
Comment (roguenet) [inline app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php:447]: Indent these two lines please.

Comment (egrojMonroy) [reply app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php]: done

File: app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php (hunk 1 above)
Comment (egrojMonroy) [reply app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php]: done

**Review by roguenet — APPROVED** [review]

**Pull request merged**